### PR TITLE
lp: add resume a job example

### PR DESCRIPTION
**Pull request by jcdietrich**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/lp.md [hunk 1]
```diff
@@ -21,3 +21,7 @@
 - Print only certain pages to the default printer (print pages 1, 3-5, and 16):
 
 `lp -P 1,3-5,16 {{path/to/filename}}`
+
+- Resume printing a job:
+
+`lp -i {{job id}} -H resume`
```
Comment (Aracki) [inline pages/common/lp.md]: I would put {{`job_id`}} to follow convention.

**Comment by sbrl** [comment]
Are we ok to merge, @Aracki?

**Comment by sbrl** [comment]
Merging, since @Aracki's comment appears to have been dealt with and we've got 3 separate approvals.

**Comment by Aracki** [comment]
@sbrl All good 👍

**Pull request merged**